packages/fx-core/tests/plugins/resource/appstudio/spfx-resources/appPackage/manifest.template.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
What about removing the legacy manifest.local/remote.template.json files if unused anymore?

Reply:
Thanks for the advice, I will work on this in the next pr.